ttt/console.py

```diff
@@ -1,24 +1,31 @@
 from ttt.consoleio import ConsoleIO
+from ttt.messages import GAME_WON_MESSAGE
+from ttt.messages import GAME_TIED_MESSAGE
 
 
 class Console:
     def __init__(self, io=ConsoleIO()):
         self._io = io
 
-    def get_console_io(self):
-        return self._io
+    def render_board(self, game):
+        board_string = self._build_board_output(game.get_board_state())
+        self._io.print_output(board_string)
 
-    def render_board(self, board_state):
```

Comment:
I prefer this signature to the new one. What benefit has changing the parameter to game given? What might it encourage violation of?